DataFormats/Math/interface/angle_units.h

```diff
@@ -11,11 +11,11 @@ namespace angle_units {
   namespace operators {
 
     // Angle
-    constexpr long double operator"" _pi(long double x) { return x * piRadians; }
```

Comment:
Shouldn't these lines be changed also?
```
  constexpr long double piRadians(M_PIl);              // M_PIl is long double version of pi
  constexpr long double degPerRad = 180. / piRadians;  // Degrees per radian
```

Reply:
I would suggest to have the two variant with two different names and use inside the operator"" the long double one
(the compiler may optimize, still)

Reply:
agree